PaulKariukiRimiru/Team-timer
# Implement toast notifications in the app

**What does this PR do?**
Implements toast notification in the app
**Description of Task to be completed?**
- Implement toast notification in the app

**How should this be manually tested?**
After cloning the app, run by running `npm start`
- Add a new team member (or remove an existing one) and you should see a toast notification

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**

N/A

**Screenshots (if appropriate)**
<img width="1440" alt="screenshot 2018-10-17 at 12 28 11 pm" src="https://user-images.githubusercontent.com/24798364/47083392-51475200-d208-11e8-8f1e-43f1d4393a86.png">


**Questions:**
N/A
